src/logrotate.conf

```diff
@@ -4,7 +4,7 @@
     compress
     sharedscripts
     postrotate
-        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw || true
+        systemctl reload ceph-mon@*.service ceph-mgr@*.service ceph-mds@*.service ceph-osd@*.service ceph-fuse@*.service ceph-radosgw@*.service || killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw || true
```

Comment:
make sense, but it will make the script more complex.
I found pkill is included in centos minmal, maybe use pkill as backup is better?